consul/template-tool/files/base_config.hcl

```diff
@@ -9,12 +9,13 @@
 {%- set vault = salt['pillar.get']('consul_template:vault', False) %}
 {%- set vault_url = salt['pillar.get']('consul_template:vault_url', 'https://vault.service.consul:8200') %}
 {%- set vault_token = salt['pillar.get']('consul_template:vault_token', 'VAULT_TOKEN') %}
+
 consul = "{{ consul_addr }}"
-token = "{{ consul_token }}" {# May also use the env var CONSUL_TOKEN #}
-retry = "10s"
+#token = "{{ consul_token }}" {# May also use the env var CONSUL_TOKEN #}
+#retry = "10s" {# doesn't work as HCL file format TODO: try as json file #}
```

Comment:
where possible, we want to stick with the HCL format (over JSON). Has the `retry` parameter been deprecated or renamed?

Reply:
what do you think open an issue to address this specific case? this wasn't the root of the problem that prevent initialize the monitor for consul-template

Reply:
OK, so the config needs to be fixed and you will do it in a separate PR, I follow. Thanks for explaining.